lib/rules/prefer-use-template-ref.js

```diff
@@ -18,13 +22,13 @@ module.exports = {
     type: 'suggestion',
     docs: {
       description:
-        'require using `useTemplateRef` instead of `ref` for template refs',
+        'require using `useTemplateRef` instead of `ref/shallowRef` for template refs',
       categories: undefined,
       url: 'https://eslint.vuejs.org/rules/prefer-use-template-ref.html'
     },
     schema: [],
     messages: {
-      preferUseTemplateRef: "Replace 'ref' with 'useTemplateRef'."
+      preferUseTemplateRef: "Replace 'ref/shallowRef' with 'useTemplateRef'."
```

Comment:
Can you please use a [message placeholder](https://eslint.org/docs/latest/extend/custom-rules#using-message-placeholders) to report either `ref` or `shallowRef` please? Please also adjust the tests then.

Reply:
Done!